source/docs/software/wpilib-tools/robot-simulation/index.rst

```diff
@@ -2,9 +2,13 @@ Robot Simulation
 ================
 
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
 
-   video-walkthrough
```

Comment:
Was the video walkthrough deleted?

Reply:
no you can see it was just added to the end along with the physics page.  Reasons for this are because I wanted to keep all the UI pages together.  Additionally the video walkthrough is outdated in some respects and IMO eclipsed by the tutorial that we have.  I think some people see a video at the top and look at that rather than keep reading and IMO that is wrong for learning about this.

Reply:
Perhaps we should just remove it?